tests/admin-unlock/main.yml

```diff
@@ -54,7 +64,14 @@
     tests: []
 
   tasks:
+    - name: Set logging
```

Comment:
I don't know if it matters, but maybe we should make the setup phase a `pre_tasks`?

Reply:
I've always used pre_tasks and post_tasks with roles because you could not mix tasks and roles.  Usually it is just to setup some variables used in the test.  Since everything is a task now, I'm not sure if it is necessary.  The only benefit I can see is that it is more explicit in the order of the tasks.

Reply:
OK, just wanted to have that conversation.